sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/samples/count/CountSamplesOutput.groovy

```diff
@@ -0,0 +1,25 @@
+package life.qbic.business.samples.count
+
+/**
+ * <b>Output interface for the {@link CountSamples} feature</b>
+ *
+ * <p>Publishes results from counting samples for one project</p>
+ *
+ * @since 1.0.0
+ */
+interface CountSamplesOutput {
+
+    /**
+     * To be called when the execution of the samples counting failed
+     * @param reason the reason why the samples retrieval failed
+     * @since 1.0.0
+     */
+    void failedExecution(String reason)
+
+    /**
+     * To be called after successfully counting samples for the provided code.
+     * @param number of all samples and samples that have been received by the lab
+     * @since 1.0.0
+     */
+    void countedSamples(int allSamples, int receivedSamples)
```

Comment:
```suggestion
    void countedReceivedSamples(int allSamples, int receivedSamples)
```

Reply:
I think this is not necessary. You have the information in the JavaDoc and the parameter name.

Reply:
But in the place were the method is called its more explicit what happens.

Reply:
See above: we count the received as well as the total number of samples to be able to display the the fraction.